build.cake

```diff
@@ -101,6 +119,68 @@ Task("Info")
     Information("Building using {0} version of Cake", cakeVersion);
 });
 
+Task("BuildDocs")
+.IsDependentOn("Build")
+.Does(() =>
+{
+    StartProcess(Context.Tools.Resolve("docfx") ?? Context.Tools.Resolve("docfx.exe"), 
```

Comment:
Question: What causes docfx to be available in the Tools context?

Reply:
This line does it https://github.com/okta/okta-auth-dotnet/pull/45/files#diff-4d3634747586b964df9f7712bc3ec669R5